### DISCUSSION TITLE
Failed to archive

### DISCUSSION THREAD

**Discussion by rares-lupascu** [codemagic-ci-cd/codemagic-docs#1470] [Q&A] [answered]
I'm able to successfully build this locally.

in codemagic i get
```
Failed to archive /Users/builder/clone/ios/xxx.xcworkspace
```
build number: 640ed9b956eba88667b10cbe

tried multiple things over the past days ... managed to maxout my free plan minutes :)

**Comment by icarusdust** [comment, marked as answer]
Hi @rares-lupascu, you are getting:

```
The following build commands failed:
	Run custom shell script 'Run Script'
(1 failure)
```

It means that executing your custom script in Xcode >> Target >> Build Phases >> Run Script is throwing this error. You need to check what custom shell script is there and correct it.

**Reply by rares-lupascu** [reply]
u were right ... but where in the logs could i have seen the error ... did i miss it?

**Reply by hkeithk** [reply]
@rares-lupascu i removed my runscript but still seeing this issue, any insights on what caused your custom runscript to fail?

**Reply by rares-lupascu** [reply]
@hkeithk my runscript was related to copying .env.production to .env so that the each scheme would use it's own env file ... this way i could have dev / staging / production (similar to flavours on android) ... i removed everything related to that script and now it's working
